### PR TITLE
Updated iOptron README

### PR DESCRIPTION
**Pull request by joeytroy**
I updated the iOptron README and also added some screenshots as well. Added information on how to pull the port number from the hand controller and to make sure to use the proper protocol version. Also verified testing with the HEM27.

### CODE REVIEW

File: indigo_drivers/mount_ioptron/README.md [hunk 1]
```diff
@@ -20,11 +20,15 @@ indigo_server indigo_mount_ioptron
...
+
+![Alt text](ioptronsettings.jpg)
 
 ## Status: Stable
 
 Driver is tested with:
-* CEM60-EC
-* iEQ 45 Pro
-
+* CEM60-EC | Default Port 4030
```
Comment (rumengb) [inline indigo_drivers/mount_ioptron/README.md:36]: I do not think the default port should be listed in Driver status section. It should probably be moved in the Comments.

File: indigo_drivers/mount_ioptron/README.md [hunk 2]
```diff
@@ -20,11 +20,15 @@ indigo_server indigo_mount_ioptron
 
 ## Comments
 
-Use URL in form ieq://host:port to connect to the mount over network (default port is 4030).
+Use URL in form ieq://host:port to connect to the mount over network. The default port can be found through the hand controller going to Settings < Wi-Fi Options < Wireless Status
+
+**EXAMPLE OF SETTING IN INDIGO**
+
+![Alt text](ioptronsettings.jpg)
```
Comment (rumengb) [inline indigo_drivers/mount_ioptron/README.md:27]: I think screenshots do not add any value as different clients show it differently and they do not show the full path to the property. It is better to descibe it in a single sentence.

File: indigo_drivers/mount_ioptron/README.md [hunk 3]
```diff
@@ -20,11 +20,15 @@ indigo_server indigo_mount_ioptron
 
 ## Comments
 
-Use URL in form ieq://host:port to connect to the mount over network (default port is 4030).
+Use URL in form ieq://host:port to connect to the mount over network. The default port can be found through the hand controller going to Settings < Wi-Fi Options < Wireless Status
```
Comment (rumengb) [inline indigo_drivers/mount_ioptron/README.md:23]: It is better to use "->" instead of "<" to show the option path.

**Review by rumengb — CHANGES_REQUESTED** [review]

**Comment by joeytroy** [comment]
@rumengb I will clean it up and push again.

**Pull request closed without merging**